scripts/helpers/tizen_gdbserver_run.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+SDB="$TIZEN_SDK_ROOT/tools/sdb"
+PORT=9999
+RESULT_MODE="debug"
+
+function help() {
+    cat <<EOF
+./$(basename "$0") --app-name=APP_NAME -- [APP_ARGUMENTS]
+EOF
+    exit
+}
+
+POSITIONAL_ARGS=()
+APP_ARGS=""
+while [[ $# -gt 0 ]]; do
+    case $1 in
+        --app-name)
+            APP_NAME="$2"
+            shift
+            shift
```

Comment:
```suggestion
            shift 2
```
In other places as well